tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2931,13 +2932,19 @@ def rgb_to_yiq(images):
   Outputs a tensor of the same shape as the `images` tensor, containing the YIQ
   value of the pixels.
   The output is only well defined if the value in images are in [0,1].
+  
+  Usage Example:
+    >>> x = tf.constant([[[1.0, 2.0, 3.0]]])
```

Comment:
Please add a newline before and after the doctest.

Also indent it at the same level as `Usage example:`